Mathlib/MeasureTheory/Integral/Average.lean

```diff
@@ -29,11 +29,6 @@ The average is defined as an integral over `(μ univ)⁻¹ • μ` so that all t
 integrals work for the average without modifications. For theorems that require integrability of a
 function, we provide a convenience lemma `MeasureTheory.Integrable.to_average`.
 
-## TODO
-
-Provide the first moment method for the Lebesgue integral as well. A draft is available on branch
-`first_moment_lintegral` in mathlib3 repository.
-
```

Comment:
Is this no longer relevant? Which theorem fixes this TODO?

Reply:
`measure_le_setLaverage_pos` and around

Reply:
Could you please move it to an entry in the module docstring?